src/sentry/integrations/slack/notify_action.py

```diff
@@ -57,11 +58,27 @@ def clean(self):
             )
             # default to "#" if they have the channel name without the prefix
             channel_prefix = self.data["channel"][0] if self.data["channel"][0] == "@" else "#"
-            channel_id = self.data["input_channel_id"]
 
         cleaned_data = super().clean()
 
         workspace = cleaned_data.get("workspace")
+
+        if channel_id:
+            try:
+                validate_channel_id(
+                    self.data.get("channel"),
+                    integration_id=workspace,
+                    input_channel_id=channel_id,
+                )
+            except Exception as e:
```

Comment:
Is every exception that can happen here actually a validation error? What if we get a connection error, or something along those lines?